### ISSUE TITLE
Accessibility issue when using keyboard (Quick Start)

### ISSUE THREAD

**Issue by chinmaydas23** [SAP/fundamental-ngx#10365]
####Is this a bug, enhancement, or feature request?
Accessibility Issue

####Briefly describe your proposal.
Observed Behavior:
The tiles for Quick Start section cannot be accessed using keyboard.

Expected Behavior:
All tiles in the quick start section should be accessible using keyboard.

####Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
angular - 15
fundamental ngx - 0.43.3

####If this is a bug, please provide steps for reproducing it.
Go to : https://spa-registry-dev-consumer-eu.dev-eu10.build.cloud.sap/lobby
Make sure the "Quick Start" section is shown, not hidden. Navigating to each tile in the quick start section using keyboard only is not possible as the whole section only gets selected and pressing of tab button just moves the focus on to next element i.e the table view component. We want to trap focus on each quick start tile and access them using the keyboard only. 
####Please provide relevant source code if applicable.
####Is there anything else we should know?
Second Image shows that  the button which helps scrolling of the tiles can be accessed using keyboard and works fine. 
<img width="949" alt="Screenshot 2023-08-21 014557" src="https://github.com/SAP/fundamental-ngx/assets/101092848/92eaf0c2-a371-49b7-b27a-5ad1d74c9810">
<img width="945" alt="Screenshot 2023-08-21 014926" src="https://github.com/SAP/fundamental-ngx/assets/101092848/48d3db8e-e702-40bb-b03e-7e2f76bf2cef">

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

I have no access to the app. Could you please give a place where I can have a look?
I checked the carousel here with cards inside and the focus is transferred to the header of the card: https://ng-15-downport--fundamental-ngx.netlify.app/#/core/carousel

Best Regards,
Nikolay

**Comment by droshev** [comment]
@chinmaydas23 do you think you can provide a stackblitz example?

**Comment by shawchandankumar** [comment]
@droshev Currently in our application there is no header in the card but only the card-content and we also don't have any focusable element inside the card-content. We want to get the whole card focused, so will that be possible?

**Comment by droshev** [comment]
@shawchandankumar Can you create a simple stackblitz with the problem you have?

**Comment by shawchandankumar** [comment]
@droshev https://ng-15-downport--fundamental-ngx.netlify.app/#/core/carousel#multi-item-active
Here we want the focus to be on the card, now it's not coming on the card, rather it's focus is on the card items

**Comment by droshev** [comment]
@shawchandankumar in that example focus on the card header too. You don't want to have nesting focus

**Comment by droshev** [comment]
I think you would need to use the [cards](https://fundamental-ngx.netlify.app/#/core/card) for that purpose
